worlds/stardew_valley/logic/skill_logic.py

```diff
@@ -15,7 +15,7 @@
 from ..data.harvest import HarvestCropSource
 from ..mods.logic.magic_logic import MagicLogicMixin
 from ..mods.logic.mod_skills_levels import get_mod_skill_levels
-from ..stardew_rule import StardewRule, True_, False_, true_
+from ..stardew_rule import StardewRule, true_, True_, False_

```

Comment:
Do we really need to import both variations of true_ ?

Reply:
No, but that would require refactoring to remove one of the true_... 

![image](https://github.com/user-attachments/assets/e1731be0-b681-4800-8546-d116f7ab328a)

Reply:
True, we can open another PR for it once this is merged
![image](https://github.com/user-attachments/assets/d4689b59-5ed2-41a9-a320-4cad1a7aae96)

Reply:
Guys pls LOL
Now I wanna make y'all change this just on PRINCIPLE